.github/workflows/linux_builds.yml

```diff
@@ -14,7 +14,7 @@ concurrency:
 
 jobs:
   build-linux:
-    runs-on: "ubuntu-20.04"
+    runs-on: "ubuntu-24.04"
```

Comment:
Actually we might want to stick to an older version for Linux and Server builds so the artifacts are somewhat portable, 24.04 might be a bit too cutting edge for people who want to test PR artifacts.